### PR TITLE
Rework stboot flags to utilize stlog properly

### PR DESCRIPTION
**Pull request by ChriMarMe**
Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: stlog/log.go [hunk 1]
```diff
@@ -45,6 +47,7 @@ type levelLoger interface {
 	warn(format string, v ...interface{})
 	info(format string, v ...interface{})
 	debug(format string, v ...interface{})
+	getLogLevel() LogLevel
```
Comment (jensdrenhaus) [inline stlog/log.go:50]: go getter usually has no get, `logLevel()`

**Review by jensdrenhaus — COMMENTED** [review]
Great!
I gues we should bring in #56 first?

**Comment by ChriMarMe** [comment]
Yes, #56 comes first.

**Review by jensdrenhaus — COMMENTED** [review]
The new log level getters should be easy to cover in the `log_test.go` and std_test.go`

File: stlog/syslog_test.go [hunk 2]
```diff
@@ -41,6 +41,29 @@ func TestKernelLogger(t *testing.T) {
 			input: "LogLevel invalid",
 		},
 	} {
+		t.Run(tt.name+" Kernel Logger", func(t *testing.T) {
```
Comment (jensdrenhaus) [inline stlog/syslog_test.go:44]: Remove equivalent t.Run in line 67ff ?

**Review by jensdrenhaus — CHANGES_REQUESTED** [review]

File: stlog/syslog_test.go (hunk 2 above)
Comment (ChriMarMe) [reply stlog/syslog_test.go]: whoopsie

**Review by jensdrenhaus — COMMENTED** [review]
Alright! Squash commits and we are done here.

**Review by jensdrenhaus — APPROVED** [review]

**Pull request merged**